### DISCUSSION TITLE
Device information missing

### DISCUSSION THREAD

**Discussion by ghost** [Koenkk/zigbee2mqtt#10764] [Devices]
Hello, 
I have been missing the page for the Moes TV01-ZB unit type for a few weeks. 
[(https://www.zigbee2mqtt.io/devices/TV01-ZB.html#moes-tv01-zb)]

It was still working before Christmas and there was still content available.

It currently displays: "404 How did we get here? Take me home"

Is there a reason for this?

**Comment by sjorge** [comment]
https://www.zigbee2mqtt.io/devices/TV02-Zigbee.html I think ZB got renamed to Zigbee in the URL, assuming this si the same device.

**Comment by ghost** [comment]
I also think that someone misunderstood and removed the URL of TV01.
But TV01 and TV02 are two completely different devices.
TV01 has much fewer SET/GET parameters and you cannot set a schedule from outside.
Who can reactivate the URL of TV01? Who is the contact person?
Zigbee2Mqtt correctly recognises the thermostat as TV01, but the link from there leads nowhere.
